osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -25,9 +25,9 @@ public sealed partial class FrameStabilityContainer : Container, IHasReplayHandl
         public ReplayInputHandler? ReplayInputHandler { get; set; }
 
         /// <summary>
-        /// The number of frames (per parent frame) which can be run in an attempt to catch-up to real-time.
+        /// The number of CPU milliseconds to spend at most during seek catch-up.
         /// </summary>
-        public int MaxCatchUpFrames { get; set; } = 5;
+        private const double max_catchup_milliseconds = 10;
```

Comment:
This value is just a ballpark good place to be. The game should always maintain at least 60 fps, and this will give enough headroom to allow that.